program-list.json

```diff
@@ -785,8 +785,8 @@
   },
   {
     "program_name": "Airtable",
-    "policy_url": "https://airtable.com/security",
-    "contact_url": "security@airtable.com",
```

Comment:
I'm not sure about this change because on one hand side the actual policy located at https://hackerone.com/airtable but on the other hand Airtable accepts reports via security@airtable.com too. Some people might prefer to report via email some via HackerOne program.

Reply:
@yesnet0 what do you think we should do in such cases?

Reply:
@nikitastupin Great question: Go with whichever primary contact channel is in the policy. @prodigysml perhaps it's time to add "contact_email" to the schema?

Reply:
@yesnet0, that seems like a good idea! Maybe making it required to have either `contact_url` or `contact_email`. Want me to try make that update?

Reply:
@prodigysml Yes please - I think @gi-el was looking at this wrt diosts as well.